### DISCUSSION TITLE
How to change the Default Weight?

### DISCUSSION THREAD

**Discussion by arjpar** [rsms/inter#702] [Q&A]
Hi, I was wondering how it could be possible to change the default weight of the normal and bold widths to 500 and 800 respectively? Would I need to use a font editor for this?

I also plan to use `otf-feature-freezer` to freeze certain features.

**Comment by kenmcd** [comment]
What are you trying to do?

Do you actually want to change the defaults in the variable font? 
Difficult - and yes, you will need an advanced font editor and know what you are doing, or know how to change the build scripts (even more difficult).
Or
Do you want to change the Medium (500) and ExtraBold (800) static fonts into your own custom RIBBI font family? And reset those to be the Regular (400) and the Bold (700) of your new family? 
Easier - but not simple. 
Would be easiest with a font editor, or something like TransType. 

Your question is unclear. Please explain.

**Reply by arjpar** [reply]
I want the Medium (500) and ExtraBold (800) static fonts to be reset to be the Regular (400) and Bold (700) (as well as corresponding Italics). Does that clear things up?

**Reply by kenmcd** [reply]
OK. Here ya go. This should give you a headstart.
[InterOTF.InterTTF.from.v4.0.(2023-11-19).release.zip](https://github.com/rsms/inter/files/15018583/InterOTF.InterTTF.from.v4.0.2023-11-19.release.zip)
Converted with TransType.
Both OTFs and TTFs now in their own RIBBI families.
Renamed the families to `InterOTF` and `InterTTF`
So you can install them without any name conflicts with original Inter or with each other.
You will need to run feature freezer on these and then you will be done.

**Reply by arjpar** [reply]
Thank you so much!
